strawberryfields/engine.py

```diff
@@ -447,21 +448,16 @@ class StarshipEngine(BaseEngine):
 
     Args:
         backend (str, BaseBackend): name of the backend, or a pre-constructed backend instance
+        polling_delay_seconds (int): the number of seconds to wait between queries when polling for
+            job results
     """
 
-    HARDWARE_BACKENDS = ("chip0",)
+    # This engine will execute jobs remotely.
+    REMOTE = True
```

Comment:
Todo: the accepted 'engine slight refactor' ADR proposes that `StarshipEngine` becomes a `HardwareEngine`/`BlackbirdEngine`, that executes both local and remote hardware simulations. That is, simulations that accept blackbird scripts, and return only samples.

`LocalEngine` will become `StateEngine`, that supports local statevector simulations.